src/components/TextInput/BaseTextInput.js

```diff
@@ -201,11 +210,10 @@ class BaseTextInput extends Component {
                         <TouchableWithoutFeedback onPress={this.onPress} focusable={false}>
                             <View
                                 style={[
-                                    styles.textInputContainer,
-                                    ...this.props.textInputContainerStyles,
-                                    this.props.autoGrow && StyleUtils.getAutoGrowTextInputStyle(this.state.textInputWidth),
-                                    !this.props.hideFocusedState && this.state.isFocused && styles.borderColorFocus,
-                                    (this.props.hasError || this.props.errorText) && styles.borderColorDanger,
+                                    textInputContainerStyles,
+
+                                    // When autoGrow is on and minWidth is not supplied, add a minWidth to allow the input to be focusable.
+                                    this.props.autoGrow && !textInputContainerStyles.minWidth && styles.mnw2,
```

Comment:
Won't this be either `false` or  the value `styles.mnw2` ? Should it be a ternary instead of `&&` ?

Reply:
It does not matter much. False will not have any effect on styles. It is simply ignored.